api/Pipfile.lock

```diff
@@ -960,6 +960,56 @@
             "markers": "python_version >= '3.7'",
             "version": "==4.1.0"
         },
+        "orjson": {
```

Comment:
@obulat if `orjson` is needed but is not listed as a dependency, it can be wiped out in a future lock or sync. I recommend adding it to the `Pipfile` as a dependency.